pegasus/sites.v3/code.org/public/css/page/music-styles.scss

```diff
@@ -94,7 +94,8 @@ section.student-projects {
 
   .text-wrapper {
     background: var(--neutral_light);
-    padding: 2rem;
+    padding-block: calc(2rem + 8px);
+    padding-inline: 2rem;
```

Comment:
What is this change?

Reply:
It increases the height of the gray box next to the iframe so they are the same height.

Reply:
Nice!